spec/features/admin_deletes_a_beer_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'spec_helper'
+feature 'As an admin, I want to be able to delete any beer', %Q{
+  [ ] I must be signed in
+  [ ] I must be on the details page for a specific beer
+  [ ] I must be able to delete my beer by clicking the delete button
+  [ ] After deleting each beer, I will be redirected to the beer index page
+} do
+  scenario 'successfully deleted a beer' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.